Source/WebSocket_Android.cpp

```diff
@@ -11,78 +11,32 @@ using namespace java::websocket;
 
 namespace UrlLib
 {
-    class WebSocket::WSImpl : public WSImplBase
+    class WebSocket::Impl : public ImplBase
     {
     public:
-        ~WSImpl()
+        Impl(std::string url, std::function<void()> onopen, std::function<void()> onclose, std::function<void(std::string)> onmessage, std::function<void()> onerror)
+            : ImplBase(url, onopen, onclose, onmessage, onerror)
```

Comment:
Use brace initialization

```suggestion
            : ImplBase{url, onopen, onclose, onmessage, onerror}
```